README.md

```diff
@@ -6,7 +6,7 @@
 
 ## Project status
 
-This project is stable, meaning the pace of new features is intentionally low. AKS Engine is maintained by teams who depend on it including [AKS Engine on Azure Stack Hub](https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-kubernetes-aks-engine-overview).
+This project is deprecated. Please consider using [Cluster API Provider Azure](https://github.com/kubernetes-sigs/cluster-api-provider-azure). This project will service major defects, but there are no planned new features. Kubernetes 1.24 will be the last version supported in AKS-Engine.
```

Comment:
should also point to AKS for users who'd rather migrate to managed k8s?

Reply:
Great suggestion!

Reply:
I was going to say exactly that. Perhaps:

This project is deprecated. Please consider either using AKS (for a managed Kubernetes option) or using [Cluster API Provider Azure](https://github.com/kubernetes-sigs/cluster-api-provider-azure) for a self-managed option. This project will service major defects, but there are no planned new features. Kubernetes 1.24 will be the last version supported in AKS-Engine.

Reply:
Updated. Thank you both.